Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9-slim
+
+LABEL maintainer="noise-kngdm <nyrsleep@protonmail.com>" version="1.0.0" 
+
+RUN groupadd -g 1000 -r music_matcher && \
+    useradd -u 1000 -m -r -g music_matcher music_matcher && \
+    apt update && apt install -y curl
```

Comment:
No es necesario instalar curl para instalar poetry. Se puede hacer con pip o wget, deberías probar esas opciones y ver si el contenedor resultante es más ligero o no.

Reply:
En un principio había pensado en instalar Poetry usando curl y el script de instalación ya que es el método recomendado por Poetry, pero tras tu comentario he estado leyendo las [razones de por qué el proyecto Poetry no recomienda los métodos de instalación alternativos](https://python-poetry.org/docs/#alternative-installation-methods-not-recommended) y he visto que, ya que en el contenedor solo usaremos una versión de Python, y no hay ningún conflicto entre las dependencias de Poetry y las del proyecto, no estaría de más instalar Poetry usando pip como sugieres, ya que el tamaño del contenedor final se reduce ~40MB:
```
mm-curl               latest            7f8501f611da   16 seconds ago       224MB
mm-pip                latest            16781a0fe7cc   About a minute ago   186MB
```
Además también disminuye el tiempo que tarda la build del contenedor:
```
➜  music-matcher git:(Objetivo-5) ✗ time docker build . -t mm-pip --no-cache -q
sha256:3472ee0d878f09b16e9e77c38c8614aa6998794db63c7b45e5bc8fb3e99ba230
docker build . -t mm-pip --no-cache -q  0.06s user 0.05s system 0% cpu 32.492 total
➜  music-matcher git:(Objetivo-5) ✗ time docker build . -t mm-curl --no-cache -q
sha256:f95e4ec2c65abeb236028713acaee3d5696ed04966b598e1ede5b928026e2645
docker build . -t mm-curl --no-cache -q  0.07s user 0.04s system 0% cpu 46.589 total
```
Lo he cambiado en 1f1586482cf5d7a5c1f9d569e6b826ff4192cab7. Lo dicho, gracias por la sugerencia!

Reply:
En realidad, se trata de usar herramientas que ya hay instaladas (`pip` o `wget` frente a otras que no están instaladas). No sé como lo habrás resuelto, pero lo que te ha venido a decir @Mapachana creo que va por ahí (por eso es más ligero porque no tienes que instalar más cosas)

Reply:
Lo anoto de cara al futuro, gracias por los comentarios!